.gitmodules

```diff
@@ -45,7 +45,7 @@
 [submodule "percona-toolkit"]
 	path = sources/percona-toolkit/src/github.com/percona/percona-toolkit
 	url = https://github.com/percona/percona-toolkit.git
-	branch = release-3.0.10
+	branch = release-3.2.1
```

Comment:
@percona-csalguero @BupycHuk Is it safe to use? Should we update our dependencies?

Reply:
Yes, why not?

Reply:
Should we update it there: https://github.com/percona/pmm-agent/blob/adfe6ceefa5cd8aab029efcdf6c71f1d6e748a76/Gopkg.toml#L39-L42 ? Somewhere else?

Reply:
https://github.com/percona/pmm-agent/pull/182